recipe/build.sh

```diff
@@ -1,4 +1 @@
-if [[ $(uname) != Darwin ]] ; then
-    TZ="Europe/London" Rscript ${RECIPE_DIR}/build.R
-fi
-${PYTHON} -m pip install . -vv
\ No newline at end of file
+"%PYTHON%" -m pip install . -vv
```

Comment:
This is the Windows batch variable syntax:
```suggestion
"${PYTHON}" -m pip install . -vv
```
Alternatively you can delete the `recipe/bld.bat` and `recipe/build.sh` scripts and use my other suggestion.

Reply:
Thanks, I'll delete the separate scripts and use your earlier suggestion.